db/migrate/20200414220332_rename_alignment.rb

```diff
@@ -1,8 +1,10 @@
-# This is needed to load the constants in PipelineRunStage
-require 'pipeline_run_stage'
-
 class RenameAlignment < ActiveRecord::Migration[5.1]
-  def change
-    PipelineRunStage.where(name: "GSNAPL/RAPSEARCH alignment").find_each { |u| u.update(name: "GSNAPL/RAPSEARCH2 alignment") }
+  # Using raw sql here because PipelineRunStage introduces dependencies on constants that may not be initialized
+  def up
+    ActiveRecord::Base.connection.execute("UPDATE pipeline_run_stages SET name='GSNAPL/RAPSEARCH alignment' WHERE name='GSNAPL/RAPSEARCH2 alignment'")
```

Comment:
Shouldn't it be:
`ActiveRecord::Base.connection.execute("UPDATE pipeline_run_stages SET name='GSNAPL/RAPSEARCH2 alignment' WHERE name='GSNAPL/RAPSEARCH alignment'")`